sklearn/tests/test_common.py

```diff
@@ -377,3 +378,15 @@ def test_transformer_n_iter():
 
         if hasattr(estimator, "max_iter") and name not in external_solver:
             yield check_transformer_n_iter, name, estimator
+
+def test_get_params_invariance():
```

Comment:
Is this in addition to where it already is? 
* `test_all_estimators` doesn't call `all_estimators` with the `include_other` flag so it excludes `Pipeline` and `FeatureUnion`.
* right now the test isn't written to handle other metaestimators, so either I can change the test or it can go in `test_non_meta_estimators`?

Thanks!